packages/expo-video/android/src/main/java/expo/modules/video/VideoManager.kt

```diff
@@ -82,4 +103,25 @@ object VideoManager {
       }
     }
   }
+
+  // We can't check if AndroidManifest.xml is configured properly, so we have to handle the exceptions ourselves to prevent crashes
+  fun runWithPiPMisconfigurationSoftHandling(shouldThrow: Boolean = false, ignore: Boolean = false, block: () -> Any?) {
```

Comment:
moved these two functions from the `VideoView` so that they are accessible there and because of DRY - let me know if this is not a good choice.